js/index.js

```diff
@@ -37,6 +37,140 @@ const siteContent = {
+})
+
+// CTA section
+
+let ctaImage = document.getElementById("cta-img");
+ctaImage.setAttribute('src', siteContent["cta"]["img-src"]);
+ctaImage.style.border = "3px dodgerblue solid"
+ctaImage.style.borderRadius = "50%"
+ctaImage.style.opacity = "1%";
+
+let ctaH1 = document.querySelector(".cta h1");
+ctaH1.innerHTML = "DOM<br> IS<br> AWESOME";
+
+
+
+let ctaBtn = document.querySelector(".cta button");
+ctaBtn.textContent = siteContent["cta"]["button"];
+ctaBtn.style.boxShadow = "5px 5px 5px dodgerblue";
+ctaBtn.style.borderRadius = "1rem"
+ctaBtn.style.border = "1px dodgerblue solid"
+ctaBtn.style.color = "dodgerblue"
+
+
+// ----------- an extra button trying to enable/disable an animation. It didn't work!
+
+// let nopeButton = document.createElement("button")
+// nopeButton.textContent = "Never mind";
+// nopeButton.style.color = "grey"
+
+// let cta = document.querySelector(".cta");
+// cta.append(nopeButton);
+
+
+// middle image
+
+let middleImage = document.getElementById("middle-img");
+middleImage.setAttribute('src', siteContent["main-content"]["middle-img-src"]);
+middleImage.style.borderRadius = "1rem"
+middleImage.style.border = "2px dodgerblue solid"
+
+
+// text content experiment
+
+let textContentP = document.querySelectorAll(".text-content > p");
+textContentP[0].textContent = siteContent["main-content"]["features-content"];
+textContentP[1].textContent = siteContent["main-content"]["about-content"];
+textContentP[2].textContent = siteContent["main-content"]["services-content"];
+textContentP[3].textContent = siteContent["main-content"]["product-content"];
+textContentP[4].textContent = siteContent["main-content"]["vision-content"];
+
+let textContentH4 = document.querySelectorAll(".text-content > h4");
+textContentH4[0].textContent = siteContent["main-content"]["features-h4"];
+textContentH4[1].textContent = siteContent["main-content"]["about-h4"];
+textContentH4[2].textContent = siteContent["main-content"]["services-h4"];
+textContentH4[3].textContent = siteContent["main-content"]["product-h4"];
+textContentH4[4].textContent = siteContent["main-content"]["vision-h4"];
+
+textContentH4.forEach(function (a) {
+  a.style.color = "dodgerblue";
+})
+
+// contact & footer
+
+let contactH4 = document.querySelector(".contact > h4");
+contactH4.textContent = siteContent["contact"]["contact-h4"];
+contactH4.style.color = "dodgerblue";
+
+let contactP = document.querySelectorAll(".contact > p");
+contactP[0].textContent = siteContent["contact"]["address"];
+contactP[1].textContent = siteContent["contact"]["phone"];
+contactP[2].textContent = siteContent["contact"]["email"];
+
+// footer
+
+let footer = document.querySelectorAll("footer > p");
+footer.textContent = siteContent["footer"]["copyright"];
+
+// ------------- Trying to get the coding image rotating by pressing the "Get Started" button. It's not working, but I don't know enough about JS to know why. I found this structure in a Stack Overflow post.
```

Comment:
Getting setInterval going can be really tough syntactically in the beginning for sure. It's still awesome to see you trying to stretch and do things a little out of the box.